lib/activerecord-import/import.rb

```diff
@@ -387,7 +387,19 @@ def import_helper( *args )
       end
 
       return_obj = if is_validating
-        import_with_validations( column_names, array_of_attributes, options )
+        if models
+          import_with_validations( column_names, array_of_attributes, options ) do |failed|
+            models.each_with_index do |model, i|
+              model = model.dup if options[:recursive]
```

Comment:
Duping the model was necessary when the `:recursive` option was set so that the parent model would be imported even if child associations failed validation. This is apparently the desired behavior since it is described [in this test](https://github.com/zdennis/activerecord-import/blob/master/test/support/postgresql/import_examples.rb#L108).